Dockerfileclean

```diff
@@ -0,0 +1,16 @@
+
+FROM python:3.8-slim
```

Comment:
python 3.8 has several vulnerabilities, try this one registry.redhat.io/rhel8/python-39